scripts/doc_templates/templates/protex

```diff
@@ -92,7 +92,7 @@
 
 # Keep this if you don't know what it does...
 # -------------------------------------------
-  $[ = 1;                 # set array base to 1
+#  $[ = 1;              # set array base to 1 (Removed to maintain compatability with Perl 5.30)
```

Comment:
I feel like it would be more clear just to remove this line, so I have done that.